### ISSUE TITLE
[BUG] All laps from Singapore and Qatar Grand Prix have IsAccurate=False

### ISSUE THREAD

**Issue by Casper-Guo** [theOehrly/Fast-F1#464]
### Describe the issue:

Every lap in those two events has `IsAccurate=False` because of the failed integrity check.

From the doc:

> Laps need to satisfy the following criteria to be marked as accurate:
> - not an inlap or outlap
> - set under green or yellow flag (the api sometimes has issues with data from SC/VSC laps)
> - is not the first lap after a safety car period (issues with SC/VSC might still appear on the first lap after it has ended)
> - has a value for lap time and all sector times
> - the sum of the sector times matches the lap time (If this were to ever occur, it would also be logged separately as a data integrity error. You usually don’t need to worry about this.)

The first three conditions are obviously satisfied by most laps. I manually checked the last two conditions and they are satisfied by most laps as well.

### Reproduce the code example:

```python
import pandas as pd
import fastf1 as f

qatar = f.get_session(2023, 17, 'R')
qatar.load(telemetry=False, weather=False, messages=False)
qatar_laps = qatar.laps

singapore = f.get_session(2023, 15, 'R')
singapore.load(telemetry=False, weather=False, messages=False)
singapore_laps = singapore.laps

all_laps = pd.concat([qatar_laps, singapore_laps])

all_laps.groupby("IsAccurate").size()
# output: False 2094

all_laps["LapTimeCheck"] = all_laps["Sector1Time"] + all_laps["Sector2Time"] + all_laps["Sector3Time"]
all_laps[all_laps["LapTimeCheck"] == all_laps["LapTime"]].shape
# output: (2026, 32)
```

### Error message:
```
core           INFO     Loading data for Qatar Grand Prix - Race [v3.1.2]
req            INFO     Using cached data for session_info
req            INFO     Using cached data for driver_info
req            INFO     Using cached data for session_status_data
req            INFO     Using cached data for lap_count
req            INFO     Using cached data for track_status_data
req            INFO     Using cached data for _extended_timing_data
req            INFO     Using cached data for timing_app_data
core           INFO     Processing timing data...
core        WARNING     No lap data for driver 55
core        WARNING     Failed to perform lap accuracy check - all laps marked as inaccurate.
core           INFO     Finished loading data for 20 drivers: ['1', '81', '4', '63', '16', '14', '31', '77', '24', '11', '18', '10', '23', '20', '22', '27', '40', '2', '44', '55']
core           INFO     Loading data for Singapore Grand Prix - Race [v3.1.2]
req            INFO     Using cached data for session_info
req            INFO     Using cached data for driver_info
req            INFO     Using cached data for session_status_data
req            INFO     Using cached data for lap_count
req            INFO     Using cached data for track_status_data
req            INFO     Using cached data for _extended_timing_data
req            INFO     Using cached data for timing_app_data
core           INFO     Processing timing data...
core        WARNING     No lap data for driver 18
core        WARNING     Failed to perform lap accuracy check - all laps marked as inaccurate.
core           INFO     Finished loading data for 20 drivers: ['55', '4', '44', '16', '1', '10', '81', '11', '40', '20', '23', '24', '27', '2', '14', '63', '77', '31', '22', '18']
```

**Comment by Casper-Guo** [comment]
I have a feeling this might not be a universal issue. I will put this in a discussion instead.

**Comment by theOehrly** [comment]
Actually, this is a bug. It's caused by Sainz' DNS in Qatar and Stroll's DNS in Singapore. But given that lap accuracy checks only take the laps for each driver separately into account, this should never change the result for the laps of other drivers. 
And given that in the case of a DNS the driver doesn't have any laps, this should not have any influence on the accuracy check at all.
I'll fix this once #449 is merged.
